index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
```

Comment:
dangit! Thanks Mike, I'll check out the article. It seems like everyone has it, but I figured that the simpler solution may not be the "ideal" solution :)